apps/gql/schema/resolvers/index.ts

```diff
@@ -347,5 +351,37 @@ export const resolvers = {
 
       return transformPanoComment(await actions.pano.comment.remove(id.value));
     },
+    createPostUpvote: async (_, { input }, { actions, pasaport: { session } }) => {
+      if (!session?.user?.id) {
+        return NotAuthorized();
+      }
+
+      if (!input.postID) {
+        return InvalidInput("post is required");
+      }
```

Comment:
should this be already required on graphql ?

Reply:
yep, this is defined as required in the schema, so we can remove this.